pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -0,0 +1,102 @@
+        plans: ["Standard", "Early Access"]
+      - name: Datadog
+        description: If you prefer hosted services, ship logs and metrics to Datadog and benefit from the native Kubernetes and EKS integration.
+        plans: ["Standard", "Early Access"]
+      - name: Grafana
+        description: Create beautiful, intuitive dashboards to track your application performance using Grafana, the open observability platform.
+        plans: ["Standard", "Early Access"]
+      - name: Porter
+        description: Combine the benefits of Kubernetes with the ease of Heroku using Porter, a Platform-as-a-Service for your Amazon EKS clusters.
+        plans: ["Standard", "Early Access"]
+      - name: Pluggable deployment strategies
+        description: Deploy your apps using a variety of industry standard deployment techniques, including canary releases and blue/green deploys. Institute automatic rollbacks in the event of a deployment failure or error condition.
+        plans: ["Standard", "Early Access"]
+      - name: Spring Boot sample app
+        description: A complete sample application demonstrating how to deploy and optimize Spring Boot applications.
+        plans: ["Standard", "Early Access"]
+      - name: Ruby on Rails sample app
+        description: A complete sample application demonstrating how to deploy and optimize Ruby on Rails applications.
+        plans: ["Standard", "Early Access"]
+      - name: Django sample app
+        description: A complete sample application demonstrating how to deploy and optimize Django applications.
+        plans: ["Standard", "Early Access"]
+      - name: Java sample app
+        description: A complete sample application demonstrating how to deploy and optimize Java applications.
+        plans: ["Standard", "Early Access"]
+  - title: Support
+    features:
+      - name: Community forum
+        description: Access to a community Slack channel where Gruntwork customers can ask for assistance from other community members and Gruntwork engineers
+        plans: ["Standard", "Early Access"]
+      - name: Private shared Slack Channel
+        description: A dedicated Slack channel for fast, private access to support from Gruntwork engineers
+        plans: ["Early Access"]
+      - name: SLA on response times
+        description: Gruntwork will provide SLAs to guarantee you the fastest support possible
+        plans: ["Early Access"]
+
+pricing:
+  - title: Standard
+    description: Wait until Gruntwork Enterprise EKS is generally available (GA)
+    disabled_price: "9,950"
+    price_styles: "color: red; text-decoration: line-through;"
+    image_styles: "max-width: 150px; margin-top: 18px;"
+
+  - title: Early Access
+    description: Sign up for early access to Gruntwork Enterprise EKS now!
+    price: "5,950"
+    image: /assets/img/grunty-enterprise-logo@2x.png
+    image_styles: "max-width: 150px; margin-top: 10px"
```

Comment:
A few questions on pricing:

1. Have we made it clear enough what the _on-going_ value add is that merits a recurring subscription, rather than a one-time fee? Commercial maintenance / updates should be called out; do we provide a UI for the Ops team? Or devs? Or something related to CI / CD that they use on an on-going basis? Vulnerability scanning? 
2. This pricing doesn't seem to scale with number of users... On the HIPAA page, for example, we mention "Up to 10 users" and a per-user price point after that. If we do per-user pricing, we'll have to think through the definition of "user." Is it just the ops team that touches the Terraform code? Or would the dev team that uses our reference apps and Helm charts be users too?
3. If we do add per-user pricing, we may want to set these price points a bit lower to start with. E.g., $4,950/mo for up to 10 users is the normal price point, and then $299/user/mo after... And for early access, each item can be discounted?

Reply:
> do we provide a UI for the Ops team? Or devs? Or something related to CI / CD that they use on an on-going basis? Vulnerability scanning?

As the current design is written in https://www.notion.so/gruntwork/Enterprise-Enterprise-EKS-344f903153b448d2a93fd9edc82adb42#0339cc091f3d4ceba2ef77455bc27f67, I think all of these will be local services and not tied to our servers (e.g., ArgoCD UI will be served from their EKS cluster). Which means it won't be something they would need our subscription to access. I think it will be a license violation for these tools if we tie our subscription to access them (and really against the spirit of open source software).

> This pricing doesn't seem to scale with number of users... On the HIPAA page, for example, we mention "Up to 10 users" and a per-user price point after that. If we do per-user pricing, we'll have to think through the definition of "user." Is it just the ops team that touches the Terraform code? Or would the dev team that uses our reference apps and Helm charts be users too?
> If we do add per-user pricing, we may want to set these price points a bit lower to start with. E.g., $4,950/mo for up to 10 users is the normal price point, and then $299/user/mo after... And for early access, each item can be discounted?

I personally don't think per user pricing makes sense here, primarily because everything we deploy including UI access is localized to the services we end up deploying. However, it might make sense to have per app pricing (like the heroku LP), as I believe we will need some work for setting up each app to provide a seamless experience. E.g., providing support for drop-in setup of their apps.

I think a lot of this pricing matters more for Patcher, which I presume is a separate pricing scale. Since I haven't stayed up to date on the thinking around Patcher pricing, for now I updated the table to include patcher for early access, but standard subscription requires additional cost (which is what patcher pricing is).

Reply:
UPDATE: Just took a look at patcher landing page and realized it's part of regular subscription, so removed the early access designation.

Reply:
Looking again at the pricing page, I'm again struggling to make the case from the customer's perspective of:

1. **Why is this an on-going subscription?** Right now, the only items that seem to be on-going are (a) commercial maintenance and (b) commercial support. But the price point may be higher than what the [K8S competitors](https://www.notion.so/gruntwork/Enterprise-Enterprise-EKS-344f903153b448d2a93fd9edc82adb42#00586350d14b44c28da407da00aaf82b) charge for a hosted and 24/7 supported K8S. I think given a direct comparison of "Enterprise K8S" offerings, that'll be a tough pill to swallow. A few ideas to consider:
    1. What if we charge a percentage of the AWS bill tied to the EKS resources? E.g., We ensure all the resources in this EKS offering have some sort of `gruntwork` tag, we sell through the AWS Marketplace, we use the AWS APIs to figure out how much those resources cost the customer, and we charge a 10% or 20% markup on them? Given that we're dealing with _enterprise_ customers here specifically, they may be spending millions of dollars per year... 
    1. Is there some tooling that _developers_ need on a regularly (e.g., daily basis) that we could build? E.g., Something that makes it easier to test their apps _locally_, in a local K8S-like env? So that service discovery, service mesh, data stores, etc all work as they expect? If we built something that targets devs, that they used on a regular basis, then we could charge on a per-developer basis. And as enterprise companies have WAY more devs than DevOps people, that would scale well.
    1. Is there any value you add we produce in the CI / CD pipeline? E.g., Gruntwork Pipelines to deploy the cluster itself? Or some sort of pieces that integrate with Argo CD, adding approval workflows, or security features, or similar? If so, I could see charging on a per-app basis.
    1. Should we pitch this as an add-on to the Gruntwork Subscription? CIS Compliance is already something we charge more for. Perhaps "Enterprise EKS" is another big upsell? In that case, it would follow our current pricing guidelines, with per-user pricing.
2. **Is there enough benefit here to sign up for early access?** Right now, it basically comes down to a higher level of support. I wonder if we need more? E.g., In the HIPAA experiment, we offer to deploy a CIS-compliant pre-prod EKS environment, and to deploy a HIPAA-compliant prod env when ready. We also provide a [table](https://www.gruntwork.io/hipaa-compliance-on-aws/#hipaa) showing what is already available and what is in progress. I wonder if a similar approach could work here? Early access subscribers get our _current_ EKS offering right away for a pre-prod env, and when the full CIS compliant one is available, we deploy the prod env with that?

I'm focused on this a lot because if we don't get the pricing right, we'll be missing the "viable" in MVP. We should merge the PR without resolving this, but I don't think doing user testing + ads before solving this would give us much. CC @josh-padnick for your thoughts & ideas too.

Reply:
Filed https://github.com/gruntwork-io/gruntwork-io.github.io/issues/570 and https://github.com/gruntwork-io/gruntwork-io.github.io/issues/569 and marked it appropriately with labels to indicate that this needs to be addressed pre-launch.